webpack/webpack.common.js

```diff
@@ -41,6 +42,18 @@ module.exports = () => {
             PROJECT_RELATIVE_NODE_MODULES_PATH = Path.resolve(APP_ROOT, '..', 'node_modules');
         }
 
+        const manifestPlugin = new WebpackManifestPlugin({
```

Comment:
There appears to be a large overlap between what this generates and what's already created in `webpack/webpack-stats.json`. Is there a chance to combine them?

Reply:
I might need to connect with you about that...been trying to do so but I'm struggling with getting a clean path key for each asset under webpack-stats.json "assets"